src/Processors/DataProcessor.php

```diff
@@ -261,4 +257,18 @@ protected function escapeRow(array $row)
 
         return $row;
     }
+
+    /**
+     * Whether to escape column or no.
+     * @param string $key
+     * @return bool
+     */
+    protected function shouldEscapeColumn($key)
+    {
+        if ($this->escapeColumns === '*') {
```

Comment:
We can add the logic here to figure out if added/edited columns are to be escaped.

```php
        foreach ($this->appendColumns as $column) {
            if ($column['name'] == $key && is_string($column['content'])) {
                return false;
            }
        }

        foreach ($this->editColumns as $column) {
            if ($column['name'] == $key && is_string($column['content'])) {
                return false;
            }
        }
```